utils/symbol-rawelf.c

```diff
@@ -13,6 +13,86 @@
 #include "utils/utils.h"
 #include "utils/symbol-rawelf.h"
 
+/*
+ *  ELF File Header validation logic.
+ */
+int elf_validation(struct uftrace_elf_data *elf)
+{
+	Elf_Ehdr *ehdr;
+	int eclass, data, version;
+	unsigned long mapped_saddr, addr;
+
+	ehdr = &elf->ehdr;
+	elf->have_section = false;
```

Comment:
I'd like to change it to `has_shdr` as we used the third-person form and section (content) itself is there - only section headers are missing.